src/Directory.Build.props

```diff
@@ -0,0 +1,34 @@
+<Project>
+  <PropertyGroup Condition="'$(MSBuildProjectExtension)'=='.csproj'">
+    <Copyright>Copyright © 2020 Your Company</Copyright>
+    <Authors>Your Authors</Authors>
+    <VersionPrefix>0.1.0</VersionPrefix>
+    <PackageReleaseNotes>First release</PackageReleaseNotes>
+    <PackageIconUrl>
+    </PackageIconUrl>
+    <PackageProjectUrl>
+    </PackageProjectUrl>
+    <PackageLicenseUrl>
+    </PackageLicenseUrl>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(MSBuildProjectExtension)'=='.csproj'">
```

Comment:
Why is the `Condition` necessary?

Reply:
To prevent issues caused by f# project when using `Directory.Build.props`. I had issues which I thought this was the fix. I will re-test to confirm if we still need this!